### PR TITLE
team_member_model created

### PR DESCRIPTION
**Pull request by sindhuja349**
please check once andi

### CODE REVIEW

**Comment by sindhuja349** [comment]
teammembers

File: users/models.py [hunk 1]
```diff
@@ -23,3 +23,11 @@ class profileModel(models.Model):
...
         return "{}".format(self.about)    
 
+    
+class TeamMember(models.Model):
+    is_capatain=models.BooleanField()
+    created_date=models.DateTimeField(default=timezone.now)
+    status=models.BooleanField(default=True)
+
+def __str__(self):
+    return "{}".format(self.is_capatain,self.created_date,self.status)
```
Comment (rajeshrapally) [inline users/models.py:33]: In str function we will get error
Please use it like
.format(self.is_capatain)

**Comment by nagpursai** [comment]
git switch main 
git pull 
git switch team_member_model
git merge main 
-----Conflicting resolve

**Comment by nagpursai** [comment]
image

File: users/models.py (hunk 1 above)
Comment (nagpursai) [reply users/models.py:33]: { } { } { }

**Review by rajeshrapally — APPROVED** [review]

**Review by nagpursai — APPROVED** [review]
LGTM

**Pull request merged**